infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -108,6 +117,28 @@
     @Getter
     private volatile CompletableFuture<Boolean> sequencerBootstrappedFuture;
 
+    /**
+     * Timeout before which the management agent attempts to bootstrap a NOT_READY sequencer.
+     */
+    private static final int bootstrapNotReadySequencerTimeout = 3;
+
+    /**
+     * This tuple is maintained to track the un-bootstrapped primary sequencer.
+     * The epoch at which the primary sequencer was found NOT_READY and the timeout value.
+     */
+    private Map.Entry<Long, Integer> seqNotReadyCountAtEpoch;
+
+    /**
+     * Cluster metrics collection.
+     */
+    //  Service to poll local node metrics.
+    private final ScheduledExecutorService localMetricsPollingService;
+    //  Locally collected nodeMetrics
+    private static final long localMetricsPollInterval = 1000;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)

Reply:
-> METRICS_POLL_INTERVAL